docs/getting-started-guides/docker.md

```diff
@@ -145,14 +145,14 @@ See [here](/docs/getting-started-guides/docker-multinode/deployDNS/) for instruc
 
 ### Turning down your cluster
 
-1. Delete all the containers including the kubelet:
+* Delete all the containers including the kubelet:
 
-Many of these containers run under the management of the `kubelet` binary, which attempts to keep containers running, even if they fail.
-So, in order to turn down the cluster, you need to first kill the kubelet container, and then any other containers.
-
-You may use `docker kill $(docker ps -aq)`, note this removes _all_ containers running under Docker, so use with caution.
+```shell
+docker rm -f kubelet
```

Comment:
The best way is probably to name the kubelet container `k8s_$(random-3char-hash)`, and then it'll be catched by the line below